spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -578,7 +569,7 @@ def contrainedplace(self, object_target: str = None, ee_orientation_at_grasping:
             object_target,
             proposition,
             self.spot,
-            self.pick_config.SEMANTIC_PLACE_ARM_JOINT_ANGLES,
+            self.pick_config.GAZE_ARM_JOINT_ANGLES,
```

Comment:
Does this function also use waypoint estimator? If so, we should make this the dynamic angle based on the target. @jimmytyyang for confirmation

Reply:
yes, true, it uses waypoint esitmation as well. @Achuthankrishna could you update this as well?